docker/transport/npipeconn.py

```diff
@@ -5,17 +5,17 @@
 from .. import constants
 from .npipesocket import NpipeSocket
 
-import http.client as httplib
-
 try:
     import requests.packages.urllib3 as urllib3
+    import requests.packages.urllib3.connection as urllib3_connection
 except ImportError:
     import urllib3
+    import urllib3.connection as urllib3_connection
```

Comment:
I just realized that this try/except Import Error is not needed anymore as docker-py requires requests >= 2.26.0 and 2.16.0 unvendored urllib3 in 2017. Just import what you need directly from urllib3. (This applies to other transports below.)

Reply:
Good point! I've simplified the imports in 115dbce2cf54457af256001e6d9376377758f58c. This also affects some unrelated pieces of code, I'm not sure whether that's OK for the maintainers, but let's see.